homeassistant/components/waze_travel_time/sensor.py

```diff
@@ -43,15 +43,16 @@
     DEFAULT_NAME,
     DOMAIN,
     IMPERIAL_UNITS,
+    SEMAPHORE,
 )
 
 _LOGGER = logging.getLogger(__name__)
 
-SCAN_INTERVAL = timedelta(minutes=5)
+SCAN_INTERVAL = timedelta(minutes=1)
```

Comment:
Why this reduction ?

Reply:
Leftover from testing. Thanks for catching it!

Reply:
Forgot to save before committing 😮‍💨